cwms-data-api/src/main/java/cwms/cda/formatters/json/JsonV2.java

```diff
@@ -107,4 +107,13 @@ public <T extends CwmsDTOBase> T parseContent(InputStream content, Class<T> type
             throw new FormattingException("Could not deserialize:" + content, e);
         }
     }
+
+    @Override
+    public <T extends CwmsDTOBase> List<T> parseContentList(String content, Class<T> type) {
+        try {
+            return om.readValue(content, om.getTypeFactory().constructCollectionType(List.class, type));
+        } catch (IOException e) {
+            throw new FormattingException("Could not deserialize:" + content, e);
```

Comment:
"Could not deserialize `content` as `type`"?

Reply:
added type in PR: https://github.com/USACE/cwms-data-api/pull/773